sky/data/storage.py

```diff
@@ -459,15 +459,26 @@ def _validate_source(source: str, mode: StorageMode,
                         '(try "/mydir: /mydir" or "/myfile: /myfile"). '
                         f'Found source={source}')
             # Local path, check if it exists
-            source = os.path.abspath(os.path.expanduser(source))
+            expanded_source = os.path.abspath(os.path.expanduser(source))
```

Comment:
small nit: we call this `full_source` (`full_src`) in other parts of the code. Do we want to keep it consistent?

Reply:
good point - fixed now